pkg/cli/mirror/fbc_operators.go

```diff
@@ -449,7 +360,14 @@ func addCatalogToMapping(catalogMapping image.TypedImageMapping, srcOperator v1a
 	if err != nil {
 		return err
 	}
-
+	// digest is returned from the result of copy to the disconnected registry, and unless there is
+	// an error during copy, this digest will be provided.
+	// ctlgSrcTIR.Ref.ID will not be empty for the case of a remote registry, but for oci FBC catalogs,
+	// this will always be empty.
+	// if both digest and ctlgSrcTIR.Ref.ID are empty, then there is no way of creating a accurate mapping source.
+	if digest == "" && ctlgSrcTIR.Ref.ID == "" {
```

Comment:
Nit: If the `digest` variable should always be set unless there was an error, it may be better to check for the digest variable first and error if that is not set. Might help catch issues in the case there was a uncaught error before calling this method.

Reply:
Makes sense.
Thanks @jpower432